radix-engine-tests/tests/system/threading.rs

```diff
@@ -0,0 +1,268 @@
+use radix_engine_tests::common::PackageLoader;
+use scrypto_test::prelude::*;
+
```

Comment:
Might be worth a quick comment at the top explaining that these tests mostly just demonstrate existing behaviour, so we can protect ourselves against future regressions? (rather than being concrete product requirements).

Reply:
For sure, to prevent regression and also to make sure relevant code paths are covered.